### ISSUE TITLE
Shows not appearing

### ISSUE THREAD

**Issue by zhkeyes** [alecgorge/PhishOD-iOS#135]
Shows are not appearing under artists

**Comment by buzzy1** [comment]
Yes shows are blank pages 😳 this is my only music avenue please fix.

**Comment by GratefulDiver** [comment]
Mine are blank unless I connect to wireless. 
This started about a week ago.
